web_group_expand/static/src/js/web_group_expand.js

```diff
@@ -0,0 +1,43 @@
+openerp.web_group_expand = function(openerp) {
```

Comment:
@oihane, please add that literal inside the function (http://www.w3schools.com/js/js_strict.asp)